### PR TITLE
ASoC: SOF: Intel: pci-tgl: add RPL-S support

### PR DESCRIPTION
**Pull request by gongjun-song**
Add PCI DID for Intel Raptor Lake S.

Signed-off-by: Gongjun Song <gongjun.song@intel.com>

### CODE REVIEW

File: sound/soc/sof/intel/pci-tgl.c [hunk 1]
```diff
@@ -170,6 +170,8 @@ static const struct pci_device_id sof_pci_ids[] = {
 		.driver_data = (unsigned long)&ehl_desc},
 	{ PCI_DEVICE(0x8086, 0x7ad0), /* ADL-S */
 		.driver_data = (unsigned long)&adls_desc},
+	{ PCI_DEVICE(0x8086, 0x7A50), /* RPL-S */
```
Comment (kv2019i) [inline sound/soc/sof/intel/pci-tgl.c]: Small note, can you use lower case, e.g. 0x7a50 like we have for other entries?

File: sound/soc/sof/intel/pci-tgl.c [hunk 2]
```diff
@@ -170,6 +170,8 @@ static const struct pci_device_id sof_pci_ids[] = {
 		.driver_data = (unsigned long)&ehl_desc},
```
Comment (kv2019i) [inline sound/soc/sof/intel/pci-tgl.c:170]: You can drop "RVP" in the git commit message. Just have "add RPL-S support" and " for Intel Raptor Lake S." in the commit message.

**Review by kv2019i — CHANGES_REQUESTED** [review]
Thanks @gongjun-song ! A few minor comments, see inline.

File: sound/soc/sof/intel/pci-tgl.c (hunk 1 above)
Comment (gongjun-song) [reply sound/soc/sof/intel/pci-tgl.c]: @kv2019i Thanks, I have changed code and comment.😊

**Review by kv2019i — APPROVED** [review]
Thanks @gongjun-song !

**Review by plbossart — APPROVED** [review]

**Pull request merged**